lib/cmds/ocr_cmds/get-suggestions.js

```diff
@@ -0,0 +1,191 @@
+'use strict';
+
+const querystring = require('querystring');
+const moment = require('moment');
+const { get } = require('../../api');
+const print = require('../../print');
+
+const getSuggestions = async (argv, nextPageToken) => {
+  const suggestionsQuery = querystring.stringify({
+    'pageSize': 25,
+    ...nextPageToken && { nextPageToken }
+  });
+  const suggestionsUrl =
+    `/v1/ocr/fhir/projects/${argv.projectId}/documentReferences/${argv.documentId}/suggestions?${suggestionsQuery}`;
+  const response = await get(argv, suggestionsUrl);
+  return response;
+};
+
+const loadAnalyzeSuggestions = async argv => {
+  const results = [];
+  let nextPageToken = '';
+  do {
+    const response = await getSuggestions(argv, nextPageToken);
+    const { records, nextPageToken: responseToken } = response.data || {};
+
+    results.push(...(records || []));
+    nextPageToken = responseToken
+      ? JSON.stringify(responseToken)
+      : undefined;
+  } while (nextPageToken);
+
+  return results;
+};
+
+const resolveValue = (value, resolver) => {
+  const firstIndex = value[0];
+  if (!firstIndex) return null;
+  const extractedValue = firstIndex.value;
```

Comment:
If you want to tighten this up, you could replace:
```
  const firstIndex = value[0];
  if (!firstIndex) return null;
  const extractedValue = firstIndex.value;
```
with
 ```
const extractedValue = value[0] && value[0].value;
```

Just an option, you do you